TicTacToe2/TicTacToe/Board.cs

```diff
@@ -6,86 +6,59 @@ namespace TicTacToe2.TicTacToe
 {
     class Board
     {
-        public static int Size { get; set; }
-        public static char[,] Space { get; private set; }
-
-        public bool validInput = false;
+        public int Size { get; }
+        public char[,] Spaces;
```

Comment:
Spaces should be private. I mentioned this in an earlier email, but I want to use a separate method of allowing an external class to access the state of the board.

Also, the S in spaces should be lowercase as it's a variable and not a property.

Finally, spaces should be readonly because the value of the variable should not be changed after the object is constructed. The values in the array can still be changed if it's readonly, but the spaces variable itself cannot be reassigned outside of the constructor. Again, we do not want to reassign it any value, so it should be readonly.